### ISSUE TITLE
[Feature Request]: Timeshift support and looking-glass VM for Windows 11

### ISSUE THREAD

**Issue by LunasShade** [prasanthrangan/hyprdots#822]
### Description

[Provide a clear and concise description of the feature you'd like to see added.]

### Use Case

[Describe how this feature would be useful to you or to other users of the project.]

### Proposed Solution

[If you have a specific solution in mind, describe it here. If not, you can skip this section.]

### Alternatives

[Are there any alternative solutions or workarounds that you've considered? If so, describe them here.]

### Additional Information

[Add any other information about the feature request here. For example, you might include links to similar features in other projects, or screenshots or diagrams to help explain your idea.]

**Comment by LunasShade** [comment]
It would be cool to have an alternative operating system for small work, testing and fun. This could be integrated into the waybar like My Linux For Work (https://www.youtube.com/watch?v=EujO_5KvCCo) for quick and easy switching.

**Comment by LunasShade** [comment]
Yes, sorry I accidentally pressed enter and it was posted even though I hadn't finished yet.

**Comment by kRHYME7** [comment]
Yeah timeshift is truly a game changer.💪

In me, I don't  have any clue on how to implement it so that most users will be happy. Considering the disk spaces and also touching the system.


But about the VM part that's awesome! 
I'll try to implement this on 
https://github.com/kRHYME7/Hyprdots-ctl.

**Comment by Katrovsky** [comment]
For Arch Linux, there is a `yabsnap` written specifically for it. It is a console utility that is easily configurable.
https://github.com/hirak99/yabsnap
